src/views/shared/Footer.js

```diff
@@ -72,6 +76,12 @@ function (Okta) {
     },
     toggleLinks: function (e) {
       e.preventDefault();
+      var revertedVal = !(
+        this.$('.js-help').attr('aria-expanded') === true ||
+        this.$('.js-help').attr('aria-expanded') === 'true'
+      );
+
+      this.$('.js-help').attr('aria-expanded', revertedVal);
```

Comment:
this works. But at this point the animation is about to start.. perhaps we can change the `aria-expanded` value once the toggle animation finishes? 
```
this.$('.js-help-links').slideToggle(200, function() {
  var expanded = this.$('.js-help').is(':visible');
  this.$('.js-help').attr('aria-expanded', expanded);
});
```

Reply:
If pick this, so the first question isn't relevant. So do you think it's fine to toggle the state after the animation, not before?

Reply:
I think it is fine to change the state after the animation. It is only fully expanded or hidden when the animation is complete.